bazel/python_downloads.bzl

```diff
@@ -0,0 +1,63 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+limited_api_build_file = """
+cc_library(
+    name = "python_headers",
+    hdrs = glob(["**/Include/**/*.h"]),
+    strip_include_prefix = "Python-{}/Include",
+    visibility = ["//visibility:public"],
+)
+"""
+
+def limited_api_download(version, sha256):
+    http_archive(
+        name = "python-{}".format(version),
+        urls = [
+            "https://www.python.org/ftp/python/{0}/Python-{0}.tgz"
+            .format(version)],
+        sha256 = sha256,
+        build_file_content = limited_api_build_file.format(version),
+        patch_cmds = [
+            "echo \#define SIZEOF_WCHAR_T 4 > Python-{}/Include/pyconfig.h"
```

Comment:
This is where I'm using string escaping to get `#define`. Without escaping this produces an error.

Reply:
I was able to get this to work with:

```
        patch_cmds = [
            "echo '#define SIZEOF_WCHAR_T 4' > Python-{}/Include/pyconfig.h"
            .format(version)]
```

Reply:
Changed to that!